pkg/catalog/catalog.go

```diff
@@ -237,12 +321,19 @@ func getBuildersFromImageStreams(imageStreams []imagev1.ImageStream, imageStream
 
 			}
 
-			catalogImage := CatalogImage{
-				Name:           imageStream.Name,
-				Namespace:      imageStream.Namespace,
-				AllTags:        allTags,
-				NonHiddenTags:  getAllNonHiddenTags(allTags, hiddenTags),
-				imageStreamRef: imageStream,
+			catalogImage := ComponentType{
+				TypeMeta: metav1.TypeMeta{
```

Comment:
should this meta structs be extracted in a function as we have been doing for Machine Readable

Reply:
Nope, we shouldn't have it in the function, should be within the code instead.

Reply:
Cool